main.tf

```diff
@@ -43,3 +43,100 @@ module "vpc" {
 
 
 ##############################################################################
+
+
+##############################################################################
+# Create CBR prewired rules for VPC -> COS (scoped to resource group)
```

Comment:
You might want to move this heading to just above the `vpc_to_cos_cbr_rule` section, and change this heading here to `Create CBR zone for all VPCs`